### PR TITLE
Adding l1_loss to paddle.nn.functional.loss

### LINKED ISSUE #19271: l1_loss

### PR DESCRIPTION
**Pull request by ZakariaR1ad**
Close #19271

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Comment by Madjid-CH** [comment]
> If you are working on an open task, please **edit** the PR description to link to the issue you've created.
> 
> For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).
> 
> Thank you 🤗

oooh sorry about that, I'll fix it right away.

**Comment by sagewhocodes** [comment]
Hello @ZakariaR1ad , Thank you for your contribution to ivy, However there are two issues:
1. It seems the PR name doesn't say about the task you have worked, you generally write `close #issue_number` on the PR description.
2. It seems it's missing a test, would you kindly look into implementing a test alongside it? You can learn more about it [here](https://unify.ai/docs/ivy/overview/deep_dive/ivy_tests.html)

**Comment by ZakariaR1ad** [comment]
Okay, I'm on it

**Comment by ZakariaR1ad** [comment]
hey @sagewhocodes, concerning the test, should I add one in the loss tests file or somewhere else ? and can you show me an example of a test for an existing function ?

**Comment by sagewhocodes** [comment]
Hello @ZakariaR1ad , Since you are adding a functional to Ivy's paddle frontend, the tests should be written in the `ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/`, there's a `test_loss.py` file there containing tests for the loss functions.  Guide to writing tests for Ivy frontend is [here](https://unify.ai/docs/ivy/overview/deep_dive/ivy_frontends_tests.html)

Also please revert any changes made to files other than the function file and the test file.

Happy Contributing 😄
